filebrowser/namers.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import unicode_literals
 import re
-from django.utils import six
+import six
```

Comment:
```suggestion

import six
```

@bluszcz Would that be better?